### ISSUE TITLE
Issues with CAPS LOCK and MACOS Mojave

### ISSUE THREAD

**Issue by harvas77** [tekezo/Karabiner-Elements#1610]
I will not mention the LED light issue, since I have seen it being mentioned by others as well. I just updated to the latest stable MacOS Mojave 10.14 and updated Karabiner to 12.1.0. I am using this great app just to eliminate the CAPS LOCK delay, and after the update this ability has gone. I cannot make it work regardless of the LED issue. Plus, I cannot update since both update options (even beta) brings up an error. Please let me know what I need to do. Much appreciated.

**Comment by harvas77** [comment]
Furthermore, I have tried the following:

"simple_modifications": [
                        {
                            "from": {
                                "key_code": "caps_lock"
                            },
                            "to": {
                                "key_code": "caps_lock"
				"hold_down_milliseconds": 0
                            }
                        }
                    ]

But keep getting:
[2018-10-21 15:47:20.023] [info] [console_user_server] Load karabiner.json...
[2018-10-21 15:47:20.024] [error] [console_user_server] parse error in /Users/harvas/.config/karabiner/karabiner.json: [json.exception.parse_error.101] parse error at 1397: syntax error - unexpected string literal; expected '}'

Any assistance will be greatly appreciated.

**Comment by harvas77** [comment]
OK, here is a small update on the above. I managed to download the latest beta update and I am now running 12.1.50. It seems that the LED issue is now fixed with this build. Furthermore, I noticed the error in my JSON script with the missing "comma" and now it load fine. BUT the delay still is not fixed. This issue is driving me crazy and I cannot type properly as I am used to this. Any help on the matter is much appreciated. Thanks again for implementing and keeping such a useful app up to date.
